### ISSUE TITLE
Priority 2 Accessibility issue: Labels did not sufficiently describe the topic or purpose of the content.

### ISSUE THREAD

**Issue by alicerichmond** [wellcomecollection/wellcomecollection.org#6743]
**What**

The ‘i’ icon button has a label of ‘Information’. These labels are duplicated for each of these buttons and also do not indicate the relation between the button and what information will be displayed.

Solution:
Ensure that all form elements have a uniquely descriptive label indicating the specific purpose of the associated form. In this instance, as each button effectively presents the same information, we would recommend removing the toggle function entirely and presenting the information below each image as plain text to resolve the issue and avoid potential confusion.

<img width="964" alt="Screenshot 2021-07-07 at 15 07 38" src="https://user-images.githubusercontent.com/7315126/124773685-12680600-df35-11eb-95a6-9d5cb0c5e9b1.png">

**Where**

URL: https://wellcomecollection.org/pages/Wvl00yAAAB8A3y8p Page: Pitch a story | Wellcome Collection

**Comment by gestchild** [comment]
I've [implemented a different solution](https://github.com/wellcomecollection/wellcomecollection.org/pull/6777), until this is discussed further with the wider team

**Comment by gestchild** [comment]
After discussion, we're happy with the implemented solution.

However, it does rely on image titles being unique, which does not always appear to be the case, e.g. https://wellcomecollection.org/articles/YNGozhIAACIAkVNF

@dannybirchall is this something you could look at please?

**Comment by dannybirchall** [comment]
Thanks @gestchild -- might be worth taking this discussion to Slack for Ben G to contribute, as he's not on GitHub.

Do image titles need to be unique per page, or across the whole site?

The image title as displayed is from the info attached to the image in the media library, yes?

I think we could probably adopt uniqueness as a rule going forward; retro-correcting would be particularly laborious with the way that images currently work

**Comment by jennpb** [comment]
@dannybirchall I'm picking up some bits and pieces from the accessibility audit...

> Do image titles need to be unique per page, or across the whole site?

Per page, as that's the context a user will be navigating with a screenreader.

> The image title as displayed is from the info attached to the image in the media library, yes?

The image title is coming from the Copyright info, e.g. `Happy endings | | | | | Laurindo Feliciano for Wellcome Collection |`

Have you adopted this uniqueness rule since July 2021, or is that still to be done?
Is it documented in any editorial/image guidance?

If so, is this ok to close?

**Comment by dannybirchall** [comment]
This is still to be done/documented. Think we can do that, but we might need a linting tool to catch old instances of duplication.

**Comment by jennpb** [comment]
Credit ambiguity addressed in #6777

Will close this now and see what next round of testing uncovers.

**Comment by jennpb** [comment]
I believe this is a human-interpreted issue. When we run another audit, if the same issue is flagged, we can discuss with DAC further. WCAG may not make a distinction between a standardised image credit and a caption. The solution from #6777 at least identifies the purpose of the `i` button as a credit.

When we write our accessibility statement (#6015), we can flag anything that doesn't strictly meet WCAG 2.1 AA guidelines, so we're calling it out as something that doesn't meet those criteria.

Carry on as you were.
